certbot/certbot/_internal/plugins/disco.py

```diff
@@ -13,6 +13,12 @@
 from certbot import interfaces
 from certbot._internal import constants
 
+try:
+    # Python 3.3+
+    from collections.abc import Mapping  # pylint: disable=no-name-in-module
```

Comment:
Disabling `no-name-in-module` is not needed here, since we are running the lint checks on Python 3.

Reply:
Thanks, updated!